contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.6.0--3.7.0.sql

```diff
@@ -10321,6 +10321,80 @@ $$
 STRICT
 LANGUAGE plpgsql;
 
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    ALTER FUNCTION sys.stuff(TEXT, INTEGER, INTEGER, TEXT) RENAME TO stuff_deprecated_3_7;
+    ALTER FUNCTION sys.stuff(ANYELEMENT, INTEGER, INTEGER, ANYELEMENT) RENAME TO stuff_any_deprecated_3_7;
```

Comment:
Why do we need the whole thing in do block?

Reply:
We don't this 3.x

Reply:
Removed `DO` block